<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempInvalidSetupDialog</class>
 <widget class="QDialog" name="TempInvalidSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="8" column="0" rowspan="2">
    <widget class="QLabel" name="lblDuration">
     <property name="text">
      <string>&amp;Длительность с</string>
     </property>
     <property name="buddy">
      <cstring>edtDurationFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QFrame" name="frmDuration">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtDurationFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>366</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDurationTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtDurationTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtDurationTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>366</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDurationDays">
        <property name="text">
         <string>дней</string>
        </property>
        <property name="buddy">
         <cstring>edtDurationTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="1" rowspan="2" colspan="2">
    <widget class="QFrame" name="frmSex">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cmbSex">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>М</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ж</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="15" column="1" colspan="2">
    <widget class="QFrame" name="frmMKB">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbMKBFilter">
        <item>
         <property name="text">
          <string>Игнор.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Интервал</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Без диагноза</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CICDCodeEdit" name="edtMKBFrom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMask">
         <string>a00.00; </string>
        </property>
        <property name="text">
         <string>A.</string>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CICDCodeEdit" name="edtMKBTo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMask">
         <string>a00.00; </string>
        </property>
        <property name="text">
         <string>Z99.9</string>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDocType">
     <property name="text">
      <string>Тип &amp;документа</string>
     </property>
     <property name="buddy">
      <cstring>cmbDoctype</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbDoctype"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblTempInvalidReason">
     <property name="text">
      <string>При&amp;чина</string>
     </property>
     <property name="buddy">
      <cstring>cmbReason</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbReason"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QCheckBox" name="chkOnlyClosed">
     <property name="text">
      <string>&amp;Только &amp;закрытые</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <item>
      <property name="text">
       <string>Врач</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblMKB">
     <property name="text">
      <string>Коды диагнозов по &amp;МКБ</string>
     </property>
     <property name="buddy">
      <cstring>cmbMKBFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblInsuranceOfficeMark">
     <property name="text">
      <string>&amp;Отметка страхового стола</string>
     </property>
     <property name="buddy">
      <cstring>cmbInsuranceOfficeMark</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="QComboBox" name="cmbInsuranceOfficeMark">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>без отметки</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>с отметкой</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="chkByPeriod">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;Если отметка &lt;span style=&quot; font-weight:600;&quot;&gt;стоит&lt;/span&gt;, то для отчёта &lt;br /&gt;выбираются документы ВУТ с &lt;span style=&quot; font-weight:600;&quot;&gt;периодом, &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;&quot;&gt;пересекающимя с периодом отчёта&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;Если отметка &lt;span style=&quot; font-weight:600;&quot;&gt;снята&lt;/span&gt;, то для отчёта &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;выбираются документы ВУТ &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;закрытые в периоде отчёта&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Отчёт по периоду случая</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="CSocStatusComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusClass</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Тип соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusType</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOldForm">
     <property name="text">
      <string>Страя форма отчета</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CSocStatusComboBox</class>
   <extends>QComboBox</extends>
   <header>Registry/SocStatusComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>chkByPeriod</tabstop>
  <tabstop>cmbDoctype</tabstop>
  <tabstop>cmbReason</tabstop>
  <tabstop>chkOnlyClosed</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>edtDurationFrom</tabstop>
  <tabstop>edtDurationTo</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>cmbSocStatusClass</tabstop>
  <tabstop>cmbSocStatusType</tabstop>
  <tabstop>cmbMKBFilter</tabstop>
  <tabstop>edtMKBFrom</tabstop>
  <tabstop>edtMKBTo</tabstop>
  <tabstop>cmbInsuranceOfficeMark</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TempInvalidSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TempInvalidSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
